src/NosCore.GameObject/Character.cs

```diff
@@ -1440,30 +1445,30 @@ WearableInstance fairy
             Morph = sp.Item.Morph;
             MorphUpgrade = sp.Upgrade;
             MorphDesign = sp.Design;
-            MapInstance.SendPacket(this.GenerateCMode());
-            SendPacket(GenerateLev());
-            MapInstance.SendPacket(this.GenerateEff(196));
-            MapInstance.SendPacket(new GuriPacket
+            await MapInstance!.SendPacket(this.GenerateCMode());
+            await SendPacket(GenerateLev());
+            await MapInstance!.SendPacket(this.GenerateEff(196));
+            await MapInstance!.SendPacket(new GuriPacket
             {
                 Type = GuriPacketType.Unknow2,
                 Value = 1,
                 EntityId = CharacterId
             });
-            SendPacket(GenerateSpPoint());
+            await SendPacket(GenerateSpPoint());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262615)